lasagne/layers/recurrent.py

```diff
@@ -438,6 +468,24 @@ class Gate(object):
     nonlinearity : callable or None
         The nonlinearity that is applied to the input gate activation. If None
         is provided, no nonlinearity will be applied.
+
+    References
```

Comment:
It was the only reference I could remember. Can you write the name of the reference? My ip is blocked?